Makefile

```diff
@@ -44,7 +44,7 @@ dimg: ## Make a Docker image for the current version
 	GOLANG_VERSION=1.9.1 ROCKSDB_VERSION=5.7.3 TAG=$(VERSION) docker-compose build
 
 drun: ## Run Faktory in a local Docker image, see also "make dimg"
-	docker run --rm -it -p 7419:7419 -p 7420:7420 contribsys/faktory:$(VERSION) -b :7419
+	docker run --rm -it -e "FAKTORY_PASSWORD=${PASSWORD}" -p 7419:7419 -p 7420:7420 contribsys/faktory:$(VERSION) -b :7419
```

Comment:
Do we need the -b :7419?  Faktory will bind to localhost:7419 and Docker forwards localhost:7419, right?

Reply:
I'm actually not sure, and it's not clear from the docs: https://docs.docker.com/engine/reference/commandline/run/#publish-or-expose-port--p-expose and https://docs.docker.com/engine/reference/run/#expose-incoming-ports. I *think* it should do the right thing if you listen on `localhost`, but it *could* be that it only forwards ports that listen on `0.0.0.0` (which presumably is what `:7419` gives you).

Reply:
I also think the `-p` rules should have `-p 127.0.0.1:74XX:74XX`, so that we don't expose the Faktory ports outside the local machine.

Reply:
Yes, it looks like you specifically need to listen on `0.0.0.0` (not `127.0.0.1`) inside the container: https://stackoverflow.com/questions/35414479/docker-ports-are-not-exposed. So the question now is what IP Faktory listens to on by default.